doc/classes/@GlobalScope.xml

```diff
@@ -2371,6 +2371,16 @@
 		<constant name="KEY_MASK_GROUP_SWITCH" value="1073741824" enum="KeyModifierMask" is_bitfield="true">
 			Group Switch key mask.
 		</constant>
+		<constant name="KEY_LOCATION_UNKNOWN" value="0" enum="KeyLocation">
+			A key which only appears once on the keyboard.
```

Comment:
Hmm, I was probably thinking that the second paragraph pretty much covers that:

> When comparing using [method InputEvent.is_match], an event which has [constant KEY_LOCATION_UNKNOWN] will match any [enum KeyLocation] on the passed event.

since `InputEvent.is_match` is the only place where `UNKNOWN` is actually used in this manner. :thinking: 

That said, it's not the only place that comparisons are exposed to the user (eg. shortcut definitions), so I could see:

> Used for keys which only appear once, or when a comparison doesn't need to differentiate the `LEFT` and `RIGHT` versions.
>
> For example, when using `InputEvent.is_match`...

(I found "keys ... for which there is no need to differentiate" was a bit non-obvious)

Thoughts?

Reply:
That looks good, let's go with that!